### PR TITLE
Remove K8s YAML template symbolic links

### PR DESCRIPTION
**Pull request by jiacheliu3**
This change removes symbolic links that link to YAML templates under `singleMaster-localJournal`. The symbolic links are now obsolete and not useful anymore.

### CODE REVIEW

**Comment by jiacheliu3** [comment]
rrrrrrrright

File: dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go [hunk 1]
```diff
@@ -194,13 +194,8 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/docker/conf/alluxio-site.properties.template",
 		"integration/docker/conf/alluxio-env.sh.template",
 		"integration/fuse/bin/alluxio-fuse",
-		"integration/kubernetes/alluxio-configmap.yaml.template",
 		"integration/kubernetes/alluxio-fuse.yaml.template",
 		"integration/kubernetes/alluxio-fuse-client.yaml.template",
-		"integration/kubernetes/alluxio-master-journal-pv.yaml.template",
-		"integration/kubernetes/alluxio-master-journal-pvc.yaml.template",
-		"integration/kubernetes/alluxio-master-service.yaml.template",
-		"integration/kubernetes/alluxio-master-statefulset.yaml.template",
```
Comment (jiacheliu3) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go]: surprised me that the `worker-daemonset.yaml.template` wasn't in all along... that's a mistake

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please.

**Pull request merged**